.github/workflows/main.yml

```diff
@@ -1479,7 +1479,7 @@ jobs:
     timeout-minutes: 60
     strategy:
       matrix:
-        java: [ '17', '22' ]
+        java: [ '17', '22', '23-ea' ]
         config: [ 'batch1', 'batch2' ]
         exclude:
           - java: ${{ github.event_name == 'pull_request' && 'nothing' || '22' }}
```

Comment:
please _replace_ 22 with 23-ea. Also in L1458. Like:

```yml
        java: [ '17', '23-ea' ]
        config: [ 'batch1', 'batch2' ]
        exclude:
          - java: ${{ github.event_name == 'pull_request' && 'nothing' || '23-ea' }}
```


The reason for that is that the job has already a `batch1`/`batch2` axis. This would create too many nodes and NB is already above the limit.

Reply:
Done, thanks.